src/ResourceManager/DataMigration/Commands.DataMigration.Test/packages.config

```diff
@@ -1,12 +1,14 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <packages>
   <package id="Microsoft.Azure.Management.Authorization" version="2.5.0-preview" targetFramework="net452" />
-  <package id="Microsoft.Azure.Management.DataMigration" version="0.1.2-preview" targetFramework="net452" />
+  <package id="Microsoft.Azure.Management.DataMigration" version="0.3.0-preview" targetFramework="net452" />
   <package id="Microsoft.Azure.Management.Resources" version="2.20.0-preview" targetFramework="net452" />
   <package id="Microsoft.Azure.Test.Framework" version="1.0.6179.26854-prerelease" targetFramework="net45" />
+  <package id="Microsoft.Azure.Test.HttpRecorder" version="1.8.1" targetFramework="net452" />
   <package id="Microsoft.IdentityModel.Clients.ActiveDirectory" version="2.28.3" targetFramework="net452" />
-  <package id="Microsoft.Azure.Test.HttpRecorder" version="1.7.0" targetFramework="net45" />
   <package id="Microsoft.Net.Http" version="2.2.28" targetFramework="net45" />
+  <package id="Microsoft.Rest.ClientRuntime" version="2.3.11" targetFramework="net452" />
+  <package id="Microsoft.Rest.ClientRuntime.Azure" version="3.3.12" targetFramework="net452" />
```

Comment:
@hitenjava please remove the references to `Microsoft.Rest.ClientRuntime` and `Microsoft.Rest.ClientRuntime.Azure` in this file as they are referenced in another file

Reply:
Done